wiki/en/Installation-for-Linux.md

```diff
@@ -25,9 +25,18 @@ We provide a repository for Debian and Ubuntu based distributions that allows au
 
 1. Open your Terminal (CTRL+ALT+T should work)
 1. *Ubuntu only* - Enable the Ubuntu "universe" repository (you can use the [GUI-based approach](https://askubuntu.com/a/148645) or [CLI-based approach](https://askubuntu.com/a/227788)).
-1. Download the repository setup script: `curl https://raw.githubusercontent.com/jamulussoftware/jamulus/main/linux/setup_repo.sh > setup_repo.sh`
-1. Make the script executable: `chmod +x setup_repo.sh`
-1. Run the setup script and install Jamulus: `sudo ./setup_repo.sh && sudo apt install jamulus`
+1. Download the repository setup script:\\
```

Comment:
I'm unfamiliar with this \\ convention. What's this?

Reply:
Basically a line break without starting a new paragraph.

Reply:
There's a convention in Markdown of two spaces at the end of a line, which may be same, but is basically invisible so if this is a visible version of that, I'm for it.